src/gtkcompletionline.c

```diff
@@ -223,11 +223,16 @@ static void gtk_completion_line_init (GtkCompletionLine *self)
    g_signal_connect(G_OBJECT(self), "scroll-event",
                     G_CALLBACK(on_scroll), NULL);
 
-   char * HOME = getenv ("HOME");
    char history_file[512] = "";
+#ifdef FOLLOW_XDG_SPEC
+   const gchar * history_dir = g_get_user_cache_dir ();
```

Comment:
I see g_get_user_cache_dir (~/.cache) a bit polluted by random files
gmrun's history is like bash history, somewhat important, it probably should go to g_get_user_config_dir ()